### PR TITLE
Fix Consenting Fetching

### LINKED ISSUE #423: Suspect the Consent display is not delay fetching as we'd like
In doing case box stuff - I would expect there to be zero calls to GetConsent unless I happen to mouse over a consent cursor. However I am noticing calls the consent api in the backend logs.

Possibly the delay load for the consent data is not delaying until the first mouse over?

(this is only a theoretical bug - I'm going off some pretty sketchy details - I've done almost no diagnosis here)

Feel free to close if I'm way wrong

### PR DESCRIPTION
**Pull request by williamputraintan**
Closes #423 

<video src="https://github.com/umccr/elsa-data/assets/61998484/5d9d7a31-0884-402a-879c-56d8dfe0b343"/>

### CODE REVIEW

File: application/frontend/src/pages/releases/detail/cases-box/consent-popup.tsx [hunk 1]
```diff
@@ -19,16 +20,27 @@ type Props = {
  * @constructor
  */
 export const ConsentPopup: React.FC<Props> = (props) => {
+  const [isConsentHover, setIsConsentHover] = useState(false);
+  console.log("isConsentHover", isConsentHover);
```
Comment (andrewpatto) [inline application/frontend/src/pages/releases/detail/cases-box/consent-popup.tsx]: !

**Review by andrewpatto — APPROVED** [review]

**Review by mmalenic — APPROVED** [review]

**Comment by andrewpatto** [comment]
Do you want to move the consent fetch over to TRPC as well?? Just whilst fixing this?

File: application/frontend/src/pages/releases/detail/cases-box/consent-popup.tsx (hunk 1 above)
Comment (williamputraintan) [reply application/frontend/src/pages/releases/detail/cases-box/consent-popup.tsx]: sorry

**Comment by andrewpatto** [comment]
Maybe it already is and what is left in the REST router is legacy.. thought I noticed some consent get stuff there

**Comment by williamputraintan** [comment]
Yes, it is in TRPC. Will remove the REST router for consent then.

**Comment by andrewpatto** [comment]
check it isn't used somewhere else I guess...

**Pull request merged**